clustersConfig.py

```diff
@@ -54,6 +54,8 @@ class ExtraConfigArgs:
 
     activation_key: Optional[str] = None
 
+    dpu_operator_branch: Optional[str] = "main"
```

Comment:
clustersConfig is a cluster-level configuration. Please use the arguments inside the extra config that deals with the dpu operator deployment. Each extra config gets a dict with args.